src/player.cpp

```diff
@@ -11000,6 +11011,15 @@ bool player::wear_item( const item &to_wear, bool interactive )
         }
     }
 
+    // Check if we don't have both hands available before wearing a briefcase, shield, etc. Also occurs if we're already wearing one.
```

Comment:
I'm assuming it's this part that caused trouble. Any idea on how best to solve that?